addons/pos_restaurant/i18n/pos_restaurant.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 15.0\n"
+"Project-Id-Version: Odoo Server 15.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-11-16 13:18+0000\n"
-"PO-Revision-Date: 2021-11-16 13:18+0000\n"
+"POT-Creation-Date: 2022-11-21 18:34+0000\n"
+"PO-Revision-Date: 2022-11-21 18:34+0000\n"
```

Comment:
Can you just remove those changes? I know it's automatically created but generally we never commit those lines

Reply:
Done